src/main/java/Rizz.java

```diff
@@ -0,0 +1,154 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Rizz {
+    private Scanner input = new Scanner(System.in);
+    private ArrayList<Task> arrList = new ArrayList<>();
+
+    private void greet() {
+        String str = "\t____________________________________________________________\n" +
+                "\tHello! I'm Rizz\n" +
+                "\tWhat can I do for you?\n" +
+                "\t____________________________________________________________\n";
+        System.out.println(str);
+    }
+
+    private void exit() {
+        String str =  "\t____________________________________________________________\n" +
+                "\tBye. Hope to see you again soon!\n" +
+                "\t____________________________________________________________\n";
+        System.out.println(str);
+    }
+
+    private void outputList() {
+        if (arrList.isEmpty()) {
+            System.out.println("\tNo items in the list.\n");
+        } else {
+            StringBuilder strBuilder = new StringBuilder();
+            for (int i = 0; i < arrList.size(); i++) {
+                strBuilder.append("\t").append(i + 1).append(". ").append(arrList.get(i)).append("\n");
+            }
+            String str = "\t____________________________________________________________\n" +
+                    "\tHere are the tasks in your list:\n" + strBuilder +
+                    "\t____________________________________________________________\n";
+            System.out.println(str);
+        }
+    }
+
+    private void markTask(int index) {
+            Task task = arrList.get(index - 1);
+            task.markAsDone();
```

Comment:
I think this should be task.setDone() due to the coding guide requirement of setters being named accordingly?